assets/javascript/app.js

```diff
@@ -0,0 +1,231 @@
+var gTopic;
+var dispTrivia = [""];
+var userChoice;
+var rRight;
+var rWRong;
+var numQAsked;
+var qIndex;
+var qTimeout;
+var rTimerInterval;
+var timeLeft;
+
+$(".sTopicSelect").click(function () {
+    gTopic = $(this).text();
+    $(".instructionModal").toggleClass("is-active");
+    startNewGame();
+});
+
+$(".rTopicSelect").click(function () {
+    gTopic = $(this).text();
+    $(".resultsModal").toggleClass("is-active");
+    startGame();
```

Comment:
did you mean `startNewGame()` ? It is not defined in the code

Reply:
yes.